0000-template.md

```diff
@@ -0,0 +1,40 @@
+# RFC-0000: Feature Name Here
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | Date of initial proposal                                                                    |
+| **Description** | One-sentence description                                                                    |
+| **Authors**     |                                                                                             |
+| **License**     | MIT                                                                                         |
```

Comment:
The writer of a feature can choose the license that applies to their feature?!

Reply:
Well, it wasn't the intention to set it up that way, moreso to have it in the RFC text somewhere so the RFCs are licensed in a standalone way in case we need to move this repo off of GitHub at some point. Not to accept RFCs without the MIT license.

Reply:
From a legal perspective, I don't know either. MIT does feel somewhat strange in that it is a software license and this is not necessarily software.

Rust-lang RFCs use MIT and they have lawyers, so it's probably good enough.

Reply:
I think the template doesn't need to specify the license as long as the repo itself has the correct license ([see, for example](https://github.com/rust-lang/rfcs/blob/master/0000-template.md)). People fork the correct license anyway, and you see if they removed it when they created the PR.  Regarding the license for the repo, I would recommend using [CC0 1.0 Universal](https://github.com/ethereum/EIPs/blob/master/LICENSE.md). But I'm not a lawyer. It's just based on the fact that similar initiatives use it, and it seems to make sense to me.

Reply:
In the case of an RFC the Unlicense might be a proper option. Option options include Creative Commons, the GNU Free Documentation License GFDL, the MIT License for Documentation, and the Apache License 2.0 for Documentation.